### ISSUE TITLE
Opening a binary with block sizes exceeding r2's defaults segfaults cutter

### ISSUE THREAD

**Issue by miyoyo** [radareorg/cutter#337]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Debian x86_64
| File format of the file you reverse (mandatory)      |  C++ Compiled w/ GCC ELF.
| Architecture/bits of the file (mandatory)            | Cortex A9 - ARMv7 32bits
| Cutter and r2 version, See About Cutter (mandatory)         | Version 1.2 / Using r2-2.2.0-554-g53df70fbd
| Show version information **full output** (See About Cutter) in a pastebin document (mandatory) | https://pastebin.com/6fE9pBbn

### Expected behavior
Load the binary
Analysis begins without crashing
### Actual behavior
Analysis crashes
### Steps to reproduce the behavior 
I cannot share the binary as it is a part of a device's firmware, however It can easily be downloaded on the internet.
In Kobo-Update-4.7.10413.zip > KoboRoot.tgz > usr > local > Kobo > libnickel.so.1.0.0

### Additional Logs, screenshots, source-code,  configuration dump, ...
![image](https://user-images.githubusercontent.com/3525610/36545406-18207e40-17e9-11e8-826d-f140f2d5673c.png)
Radare2 was compiled yesterday evening (GMT+1) from their github and no configuration was changed
Cutter was build this morning around 1 AM (GMT+1) with Qt5.9.1 downloaded from the website

**Comment by Maijin** [comment]
Related https://github.com/radareorg/cutter/issues/285

**Comment by thestr4ng3r** [comment]
Seems to work for me. At least it has been running for 20 minutes now. Could you please try again and provide a stacktrace?
md5 for the libnickel.1.0.0 in the archive I have downloaded is 98fa9242d3a7bf2a4da9e504805b099e. Is that the correct one?

**Comment by xarkes** [comment]
ping @miyoyo

**Comment by miyoyo** [comment]
https://pastebin.com/Y0c4hZke

Sorry for the delay, but do you mean this?

**Comment by thestr4ng3r** [comment]
Exactly, that is really helpful. Can you please also check that the md5 I mentioned is the correct one for libnickel.so.1.0.0? 98fa9242d3a7bf2a4da9e504805b099e

**Comment by miyoyo** [comment]
`
        case ARM_INS_BFI:
        {
                ut64 mask = bitmask_by_width[IMM (3) - 1];
                ut64 shift = IMM (2);
                ut64 notmask = ~(mask << shift);
                // notmask,dst,&,lsb,mask,src,&,<<,|,dst,=
                r_strbuf_setf (&op->esil, "%"PFMT64u",%s,&,%"PFMT64u",%"PFMT64u",%s,&,<<,|,0xffffffff,&,%s,=",
                        notmask, REG (0), shift, mask, REG (1), REG (0));
                break;
        }
`

I think it's referencing this piece of code, maybe It's broken on my end?

**Comment by xarkes** [comment]
I couldn't reproduce a crash, maybe that was fixed in a recent radare2 update?
